canister/src/api/get_utxos.rs

```diff
@@ -1192,4 +1210,40 @@ mod test {
         // Base fee + instructions are charged for.
         assert_eq!(runtime::get_cycles_balance(), 10 + 1000);
     }
+
+    #[test]
+    fn test_get_stability_count_single_block_on_height() {
+        let block = BlockBuilder::genesis().build();
+        let blocks_with_confirmations: Vec<(&Block, u32)> = vec![(&block, 1)];
```

Comment:
```suggestion
        let blocks_with_depths: Vec<(&Block, u32)> = vec![(&block, 1)];
```
(ditto elsewhere in other tests)